vignettes/teal-slice-classes.Rmd

```diff
@@ -117,9 +117,9 @@ The following `FilteredDataset` child classes are currently implemented:
 
 `FilterStates` serves two `shiny`-related purposes:
 
-- `ui/srv_add` allows adding a `FilterState` for a selected variable. The variables included in the module are the filterable column names of the provided data set. Selecting a variable adds a `FilterState` to the `reactiveVal` stored in the `private$state_list` private field. The subtype of the created `FilterState` automatically determined based on the class of the selected column.
+- `ui/srv_add` allows adding a `FilterState` for a selected variable. The variables included in the module are the filterable column names of the provided data set. Selecting a variable adds a `FilterState` to the `reactiveVal` stored in the `private$state_list` private field. The specific variation of the created `FilterState` is automatically determined based on the class of the selected column.
```

Comment:
What's wrong with "subtype"?

Reply:
I believe "Make Sure inst/WORDLIST is minimalized" was intended to mean make sure the wordlist does not contain only words that actually occur in the  package. "Subtype" is a valid term, as are supertype, subclass, and superclass.